### DISCUSSION TITLE
Game crash during creation of the world

### DISCUSSION THREAD

**Discussion by Wolfo07** [Sinytra/ForgifiedFabricAPI#90] [Q&A]
Hi.

 I've installed the differents requirements to make Fabric's mods work with forge but when I load a world the game crash. I don't know if the fabric's mods are causing this or if this is a compatible issue with forge's mods I have. 

Please help (The Fabric's mods I want are RPG Series made by daedelus_dev)

**Comment by windsoryeh** [comment]
I have the same issue with both server and client.
